flexget/plugins/filter/exists_series.py

```diff
@@ -33,7 +33,9 @@ class FilterExistsSeries(object):
                 'type': 'object',
                 'properties': {
                     'path': one_or_more({'type': 'string', 'format': 'path'}),
-                    'allow_different_qualities': {'enum': ['better', True, False], 'default': False}
+                    'allow_different_qualities': {'enum': ['better', True, False], 'default': False},
```

Comment:
Is this pep8-compliant? It looks like it's indented 1 more space than the line above it.